### PR TITLE
Update tests

### PR DESCRIPTION
**Pull request by CunliangGeng**
- Removed unnecessary folders.
- Update tests to test different MIME types

### CODE REVIEW

File: tests/rdf_format_converter.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python3
```
Comment (c-martinez) [inline tests/rdf_format_converter.py:1]: Is this script just required to convert ttl test data to all other formats?

**Review by c-martinez — CHANGES_REQUESTED** [review]
Tests are failing in travis https://travis-ci.org/github/c-martinez/FAIRDataPoint

I think Makefile replaces tabs for spaces and make does not like that. I fixed to test locally, but then tests are still failing.

File: tests/rdf_format_converter.py (hunk 1 above)
Comment (CunliangGeng) [reply tests/rdf_format_converter.py:1]: yep, `ttl` is our default format, and this script is just used to generate test sample files.

**Comment by CunliangGeng** [comment]
> Tests are failing in travis https://travis-ci.org/github/c-martinez/FAIRDataPoint
> 
> I think Makefile replaces tabs for spaces and make does not like that. I fixed to test locally, but then tests are still failing.

I'll have a look.

**Comment by CunliangGeng** [comment]
Changing space to tab for makefile works now, the test passed.

File: tests/rdf_format_converter.py (hunk 1 above)
Comment (c-martinez) [reply tests/rdf_format_converter.py:1]: Maybe you can add a one line comment explaining what the script does (and that it is for internal use mainly).

**Comment by c-martinez** [comment]
Great -- they were still breaking in my machine, but I figured out it is because my python version is too old.

**Review by c-martinez — APPROVED** [review]

File: tests/rdf_format_converter.py (hunk 1 above)
Comment (CunliangGeng) [reply tests/rdf_format_converter.py:1]: Thanks, updated!

**Review by c-martinez — APPROVED** [review]

**Pull request merged**